py/setup.py

```diff
@@ -380,7 +386,7 @@ def run(self):
     long_description=long_description,
     ext_modules=ext_modules,
     install_requires=[
-        "torch>=1.13.1",
+        "torch >=2.0.1" if not LEGACY else "torch >=1.13.0,<2.0",
```

Comment:
Did not add limit on Torch version in non-legacy installs for `main`, since active development can require frequent version updates.